.github/ISSUE_TEMPLATE/flaking-test.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Flaking Test about: Report flaky tests or jobs in TiCDC CI labels: component/test
+
+---
+
+<!-- Please only use this template for submitting reports about flaky tests or jobs (pass or fail with no underlying change in code) in TiCDC CI -->
+
+#### Which jobs are flaking:
+
+#### Which test(s) are flaking:
+
+#### Jenkins Or GitHub Actions link:
```

Comment:
Currently jenkins archive every two days, better to record the original error log

Reply:
Good catch! Addressed!